internal/website/content/howto/docstore/queries.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Run Queries"
+date: 2019-06-12T17:41:56-04:00
+draft: true
+weight: 3
+---
+
+Docstore's `Get` action lets you retrieve a single document by its primary key.
+Queries let you retrieve all documents that match some conditions.  You can also
+use queries to delete or update all documents that match the conditions.
+
+<!--more-->
+
+## Getting Documents
+
+Like [action lists]({{< ref "actions.md#actions-and-action-lists" >}}), queries are built up in a fluent style:
+
+```go
+iter := coll.Query().Where("Score", ">", 20).Get(ctx)
+```
+
+
+Just as a `Get` action returns one document, the `Query.Get` method returns
```

Comment:
How about just "returns an iterator. Repeatedly calling..."

Reply:
I want to emphasize the relationship between the `Get`s, since this was something that bothered Russ.

- action `Get` => one document
- query `Get` => several documents